.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] Linters help us now if our code is well formatted but also infroms us if it presents a bug that will stop our code from running. Linters are not 100 percent accurate but the majority of time are more helpful than not, Because of this we have to make sure we are always using a linter and reviewing our code before submitting code to our repository even more to a production stage. 

We should review why our code is failing the linters check. Please review the errors locally or on the Check Tab that appears in this Pull Request to see the output of the linters.

Reply:
:heavy_check_mark: This issue has been fixed